app_noticias/views.py

```diff
@@ -17,6 +17,11 @@ class HomePageView(ListView):
     def get_queryset(self):
         return Noticia.objects.exclude(data_de_publicacao=None).order_by('-data_de_publicacao')[:5]
```

Comment:
essas já são as cinco notícias mais recentes, não seria necessário o código do método `get_context_data()`